common/src/main/java/com/microsoft/identity/common/CodeMarkerUtil.java

```diff
@@ -0,0 +1,14 @@
+package com.microsoft.identity.common;
+
+import java.util.List;
+
+public class CodeMarkerUtil {
+    public static String getCSVContent(List<CodeMarker> codeMarkers) {
+        StringBuilder stringToWrite = new StringBuilder("TimeStamp,Marker,Time,Thread,CpuUsed,CpuTotal,ResidentSize,VirtualSize,WifiSent,WifiRecv,WwanSent,WwanRecv,AppSent,AppRecv,Battery,SystemDiskRead,SystemDiskWrite");
```

Comment:
CodeMarker needs to create this...and be aware of the order...and somehow couple this with generating the csv content

Reply:
Done. Thanks